cwms-data-api/src/main/java/cwms/cda/data/dto/TimeSeries.java

```diff
@@ -252,12 +266,12 @@ else if(format.equals("xml")) {
     @ArraySchema(
         schema = @Schema(
             name = "TimeSeries.Record",
-            description = "A representation of a time-series record",
+            description = "A representation of a time-series record in the form [dateTime, value, qualityCode]",
             type="array"
         ),
         arraySchema = @Schema(
             type="array",
-            example = "[123,54.3,0]"
+            example = "[123, 54.3, 0]"
```

Comment:
Can you change the `123` timestamp to a timestamp within the last decade?

Reply:
done